### PR TITLE
Newsletter April 2022

### PR DESCRIPTION
**Pull request by domidomi**

### CODE REVIEW

File: newsletter/2022-April-vX.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# April 2022 StyleGuide newsletter
+
+Combined update notes, summarising changes in the style guide since the last newsletter (since StyleGuide v208.2.0). Some of the key highlights include:
+
+- Storybook deployment for each PR now available
```
Comment (coderitual) [inline newsletter/2022-April-vX.md:5]: deployment and **preview**

File: newsletter/2022-April-vX.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Dialog component improvements:
+
+  - Added onItemSelect handler to dropdown #2327
```
Comment (katarzynatobis) [inline newsletter/2022-April-vX.md:9]: this doesn't belong to Dialog improvements

File: newsletter/2022-April-vX.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Updated old favicon designs / added notification favicons (#2351)
+
+- Accessibility improvements:
+
+  - A11y docs in storybook for various components
+  - Link component accessiblity improvements #2329
+  - Remove icon titles in browser tooltips #2341
+  - New tab indicator in link (#2365)
+  - Pass target to link (#2379)
```
Comment (katarzynatobis) [inline newsletter/2022-April-vX.md:29]: + separators accessibility improvements

**Review by clxandstuff — CHANGES_REQUESTED** [review]

File: newsletter/2022-April-vX.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+  - Added onItemSelect handler to dropdown #2327
+  - add test id props to dialog (#2371)
+  - Fix Dialog Remove No-Scroll Class Before the Exit Callback (#2388)
+  - Fix Undefined Window in Server-Side Rendering #2308
+  - Fix Unimplemented TransitionEnd Event #2299
+  - Prevent Dialog scrollbars from showing up during show transition #2328
+  - Adjust Dialog fullscreen animation #2339
+
+- Responsive props documentation for components
```
[the remark this answers is not in the document]
Comment (clxandstuff) [reply newsletter/2022-April-vX.md]: Its on line 18

**Review by clxandstuff — APPROVED** [review]

**Pull request merged**